organization/_posts/2018/2018-09-19-packaging.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "HSF Packaging Group Meeting #27, September 19, 2018"
+layout: meetings
+---
+
+# {{page.title}}
+
+
+Agenda
+=======
+
+[https://indico.cern.ch/event/754985/](https://indico.cern.ch/event/754985/)
+
+Participants: Graeme Stewart, Ben Morgan, Guilherme Amadio, Javier Cervantes Villanueva, Patrick Gartung, Liz Sexton-Kennedy, Ben Couturier, Marco Clemencic, Martin Ritter, Jim Amundson, Hadrian Grasland, Todd Gamblin, Oana Boeriu
+
+[Introduction - Graeme](https://indico.cern.ch/event/754985/contributions/3128664/attachments/1709906/2774762/HSF_Packaging_Group_Intro_2018-09-191.pdf)
+=====================
+- CHEP Presentations x5 on packaging topics
+- Liz asked about how IceCube discovered Spack (was it via HSF?)
+  - Not clear at the moment
+- Report to LHCC on Software
+  - Packaging included: main feedback, “when will conclusions be reached?”
+  - For us, means we need to start delivering working prototypes
+- Today: Two updates on Spack work
+
+
+
+[Spack and FCC Update - Javier](https://indico.cern.ch/event/754985/contributions/3128668/attachments/1718723/2773707/SpackHSFMeeting-SummerStudent2018.pptx.pdf)
+=============================
+- FCC Use Case:
+  - Use LCG CVMFS install (base), plus Spack "packages.yaml" describing them as “system” packages
+    - Graeme: is there a script to generate this? Yes.
+  - Allows override of LCG packages as required
+  - Layered config: FCC specific packages used first, then LCG then Spack base
+- Workflow:
+  - From CVMFS, merge LCG and FCC packages.yaml
+  - Run `spack install <metapackage>` using this, outputting binary tarballs
+  - Binary tarballs deployed/published on CVMFS, plus spack view creation
+  - Graeme: Have looked at combining Build/Publication node with Tomas Stefan (summer student), presentation is [here](https://indico.cern.ch/event/746546/contributions/3086271/attachments/1708942/2754445/Deployment_of_ATLAS_releases.pdf).
```

Comment:
Small point, but maybe rephrase this to

"Graeme: We have tested parallel publication to CVMFS from many nodes that not only performs better, but simplifies the workflow as the build node can also take care of publishing the results (see [presentation](https://indico.cern.ch/event/746546/contributions/3086271/attachments/1708942/2754445/Deployment_of_ATLAS_releases.pdf) from Tomas Stefan's summer student project)."